plugins/basic/nexus-repository-raw/src/main/java/org/sonatype/nexus/repository/raw/internal/RawProxyRecipe.groovy

```diff
@@ -121,7 +126,14 @@ class RawProxyRecipe
     Router.Builder builder = new Router.Builder()
 
     builder.route(new Route.Builder()
-        .matcher(new TokenMatcher("/{name:.+}"))
+        .matcher(new SuffixMatcher('/', false))
```

Comment:
I don't think this is quite correct - not sure this handler should be applied to proxies at all.

Consider someone pointing a raw proxy at a web server, configured with a default page of 'index.htm'. Then, a request comes into our proxy for '/'.  Normally, this would mean we go remote and store whatever we get from '/' in the cache, but  instead the IndexHtmlForwardHandler will replace the request with one for /index.html, which will go remote and return 404.

This substitution seems like something that should be handled only by 'leaf' repositories (hosted, remotes) - proxies and groups don't get involved.